src/templates/finops-hub/schema/schema_ea_normalized.json

```diff
@@ -0,0 +1,723 @@
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "CostCenter",
+          "type": "String"
+        },
+        "sink": {
+          "name": "CostCenter",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "CostInBillingCurrency",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "CostInBillingCurrency",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "costInPricingCurrency",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "CostInPricingCurrency",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "costInUsd",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "CostInUsd",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "Date",
+          "type": "DateTime"
+        },
+        "sink": {
+          "name": "Date",
+          "type": "DateTime"
+        }
+      },
+      {
+        "source": {
+          "name": "EffectivePrice",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "EffectivePrice",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "exchangeRateDate",
+          "type": "DateTime"
+        },
+        "sink": {
+          "name": "ExchangeRateDate",
+          "type": "DateTime"
+        }
+      },
+      {
+        "source": {
+          "name": "exchangeRatePricingToBilling",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ExchangeRatePricingToBilling",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "Frequency",
+          "type": "String"
+        },
+        "sink": {
+          "name": "Frequency",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "invoiceId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "InvoiceId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "InvoiceSectionId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "InvoiceSectionId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "InvoiceSectionName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "InvoiceSectionName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "IsAzureCreditEligible",
+          "type": "String"
+        },
+        "sink": {
+          "name": "IsAzureCreditEligible",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "location",
+          "type": "String"
+        },
+        "sink": {
+          "name": "Location",
```

Comment:
Should we call this ResourceLocationNormalized so it feels more closely aligned with the ResourceLocation value?

Reply:
Done